ui/src/app/components/common-env-conf-panels/environment-informed-action-panel.tsx

```diff
@@ -14,64 +16,52 @@ import { EnvironmentCostEstimator } from './environment-cost-estimator';
 import { StartStopEnvironmentButton } from './start-stop-environment-button';
 import { styles } from './styles';
 
-interface CostInfoProps {
+interface CostsDrawnFromProps {
+  usingInitialCredits: boolean;
+  userIsCreator: boolean;
   creatorFreeCreditsRemaining: number;
-  environmentChanged: boolean;
-  analysisConfig: AnalysisConfig;
-  currentUser: string;
-  workspace: Workspace;
-  isGKEApp: boolean;
+  billingAccountName: string;
 }
-const CostInfo = ({
+const CostsDrawnFrom = ({
+  usingInitialCredits,
+  userIsCreator,
   creatorFreeCreditsRemaining,
-  environmentChanged,
-  analysisConfig,
-  currentUser,
-  workspace,
-  isGKEApp,
-}: CostInfoProps) => {
+  billingAccountName,
+}: CostsDrawnFromProps) => {
   const remainingCredits =
     creatorFreeCreditsRemaining === null ? (
       <Spinner size={10} />
     ) : (
       formatUsd(creatorFreeCreditsRemaining)
     );
 
-  return (
-    <FlexRow data-test-id='cost-estimator'>
-      <div
-        style={{
-          padding: '.495rem .75rem',
-          ...(environmentChanged
-            ? {
-                backgroundColor: colorWithWhiteness(colors.warning, 0.9),
-              }
-            : {}),
-        }}
-      >
-        <EnvironmentCostEstimator {...{ analysisConfig }} isGKEApp={isGKEApp} />
```

Comment:
moved this to the main component because it's no longer always grouped with "draws from"